src/components/EventDetails/styles.js

```diff
@@ -0,0 +1,41 @@
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    marginTop: theme.spacing(9),
```

Comment:
What is this marginTop for? theme.spacing(9) is massive

Reply:
This is just so that the EventDetailsComponent is not covered by the navbar.

Reply:
Yeah this is what I meant when I said that components need to have an explicit margin at the top so that the NavBar does not hide stuff. Although I thought this was no longer needed..

Reply:
I'm 100% sure I fixed this

Reply:
EventDetails component still gets partially covered by the NavBar if I remove theme.spacing on my end though. I'm not too sure why then.

Reply:
Just kidding - looks like I never merged in the changes :)